zkevm-circuits/src/table.rs

```diff
@@ -2229,6 +2236,198 @@ impl<F: Field> LookupTable<F> for SigTable {
     }
 }
 
+/// 1. EcAdd: (arg1_rlc, arg2_rlc) + (arg3_rlc, arg4_rlc) = (output1_rlc, output2_rlc)
```

Comment:
this comment is a bit misleading as we are not really doing additions on RLCs

I suggest the following:

`if EcAdd(a, b) == c then (arg1_rlc, arg2_rlc, arg3_rlc, arg4_rlc) \mapsto (output1_rlc, output2_rlc)`

where

`arg1_rlc = rlc(a.x)`
`arg2_rlc = rlc(a.y)`
...


Similar for the rest of the comments